codefori/vscode-ibmi
# Await on event subscribers functions

### Changes
This PR fixes an issue where too many requests are made at the same time on the SSH connection when connecting to an IBM i, resulting in this error showing up:
![image](https://github.com/codefori/vscode-ibmi/assets/11096890/8e041c53-a927-4c1f-9887-eb1151ffc270)

This was due to the functions being triggered by the `Connected` event making too many requests in parallel.
Waiting for each function to end solves this problem.

### How to test this PR
- Connect to a system
- Make sure everything loads correctly (debug status, library list view, ...)
- The error message in the description must not show up

### Checklist
